libs/application-generic/src/utils/slugify-identifier.ts

```diff
@@ -0,0 +1,8 @@
+import slugify from 'slugify';
+
+export const slugifyIdentifier = (identifier: string): string => {
```

Comment:
```suggestion
export const slugify = (identifier: string): string => {
```

or `slufigyName`.

Reply:
Will do slufigyName in order to be more specific and so it won't collide with the lib function